app/src/main/java/com/eveningoutpost/dexdrip/Services/LibreWifiReader.java

```diff
@@ -422,15 +478,15 @@ private void readData() {
         // Start very simply by getting only one object and using it.
         Log.e(TAG, "readData called");
         Long LastReportedTime = 0L;
-        
+
         // TODO change that to work based on readings as well ???
         LibreBlock libreBlock = LibreBlock.getLatestForTrend(0L, JoH.tsl() + 5 * 60000); // Allow some packets from the future.
         if (libreBlock != null) {
             LastReportedTime = libreBlock.timestamp;
             Log.e(TAG, "LastReportedTime = " + JoH.dateTimeText(LastReportedTime));
 
             // jamorham fix to avoid going twice to network when we just got a packet
-            if ((new Date().getTime() - LastReportedTime) < DEXCOM_PERIOD - 2000) {
+            if ((new Date().getTime() - LastReportedTime) < 60000 - 2000) {
```

Comment:
Is `60000` arbitrary chosen? It should be a constant, shouldn't it? Also, `2000` looks arbitrary chosen, as well.

Reply:
I have replaced the 60,000 with Constants.MINUTE_IN_MS as this is what it means.
We expect a new reading every 60 seconds, so no need to check before that (probably just a waste of battery).
The 2,000 from before is 2 seconds, and it is meant to be that we allow some error tolerance with timeing, so actualy we want to check every 58 seconds.
The 2000 can be replaced with a constatnt but it will not mean match.